resources/checkDefinitions.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs').promises;
```

Comment:
Could you move this file to the other CI check scripts in the `/ci` directory, and maybe rename it to `definitionsCheck.js`?